docs/victoriametrics-cloud/account-management/registration-and-trial.md

```diff
@@ -0,0 +1,36 @@
+---
+weight: 1
+title: Registration and Trial
+disableToc: true
+menu:
+  docs:
+    parent: account-management
+    weight: 1
+---
+
+Start your registration process by visiting the [Sign Up](https://console.victoriametrics.cloud/signUp?utm_source=website&utm_campaign=docs_registration_and_trial) page.
+
+## Authentication methods
+
+VictoriaMetrics Cloud supports user registration and authentication via the following mechanisms:
+
+1. Sign up with Google.
+2. Email and password.
+
+
+## Trial period and credits
+
+After registration, every new user is granted with $200 in credits to spend during the trial period.
+The trial period starts once the first deployment is created and lasts for 30 days.
+
+In general, VictoriaMetrics Cloud billing is based on the time a deployment is running.
+This means that, during the trial period, you are welcome to start, test and delete as many VictoriaMetrics
+Cloud deployments as you wish.
+
+Adding a payment method is not required to register or make use of deployments during the trial period. Once the credits are expired,
+existing trial deployments will be automatically deleted. If you add a [payment method](https://docs.victoriametrics.com/victoriametrics-cloud/billing/#payment-methods),
+the service won't be disrupted and you will be charged on that one once the credits are exhausted.
+
+## After trial version expires
+When the trial period ends, adding a [payment method](https://docs.victoriametrics.com/victoriametrics-cloud/billing/#payment-methods) will let you continue
+using VictoriaMetrics Cloud. After the trial period expires, deployments will be stopped and deleted if no payment methods are found for your account. If you need assistance or have any questions, don't hesitate to contact our support team at support-cloud@victoriametrics.com.
```

Comment:
do we have any graceful period between stop and delete, if yes let's specify

Reply:
Good point. Thanks! It's 7 days.

```suggestion
using VictoriaMetrics Cloud. After the trial period expires, deployments will be stopped and deleted after 7 days if no payment methods are found for your account. If you need assistance or have any questions, don't hesitate to contact our support team at support-cloud@victoriametrics.com.
```